BUILD.md

```diff
@@ -65,6 +94,8 @@ The shared library in the release Nuget(s) and the Python wheel may be installed
 |Ubuntu 16.x  | YES          | YES         | Also supported on ARM32v7 (experimental) |
 |Mac OS X  | YES          | NO         |    |
 
+* Red Hat Enterprise Linux and CentOS are not supported.
```

Comment:
"Red Hat Enterprise Linux and CentOS are not supported." is outdated and can be removed?

Reply:
Sorry, I had an outdated version. Fixed issues in latest commit.